tests/unit/test_pool.py

```diff
@@ -655,193 +874,36 @@ def test_ping_oldest_stale_and_not_exists(self):
 
         self.assertTrue(SESSIONS[0]._pinged)
         SESSIONS[1].create.assert_called()
+        self.assertNoSpans()
 
-
-class TestTransactionPingingPool(unittest.TestCase):
```

Comment:
I didn't mean to remove the existing tests - TestTransactionPingingPool. Can you revert this back?
Since this was already deprecated we need not any observability related tests here.

Reply:
Okay gotcha. Sure let me make the update shortly.